tetris.py

```diff
@@ -293,20 +295,48 @@ def block_rotator(self):
 
     def full_line_clear(self):
         """
-        Clears the line once it's fully populated with blocks
+        Flashes the line once it's fully populated with blocks and clears it afterwards
         """
+        full_lines = []
         y_coordinates = [y for (x, y) in self.coord_extractor()]
         coordinates_counter = collections.Counter(y_coordinates)
         for x_line in range(game_height):
             count = coordinates_counter[x_line]
             if count == game_width:
-                # TODO: root.after() here
-                for letter, coord_list in self.landed_blocks.items():
-                    # self.landed_blocks = {letter: [(a, b) for (a, b) in coord_list if b > x_line] + [(a, b+1) for (a, b) in coord_list if b < x_line]} #Why this doesn't work?
-                    self.landed_blocks[letter] = [(a, b) for (a, b) in coord_list if b > x_line] + [
-                        (a, b + 1) for (a, b) in coord_list if b < x_line
+                full_lines.append(x_line)
+        if full_lines:
+            for flash in range(2):
+                self.flasher(full_lines, 'yellow')
+                self.canvas.update()
+                time.sleep(0.1)
+                self.flasher(full_lines, 'blue')
+                self.canvas.update()
+                time.sleep(0.1)
+            self.canvas.delete('flash')
+        for x_line in full_lines:
+            for letter, coord_list in self.landed_blocks.items():
+                # self.landed_blocks = {letter: [(a, b) for (a, b) in coord_list if b > x_line] + [(a, b+1) for (a, b) in coord_list if b < x_line]} #Why this doesn't work?
+                self.landed_blocks[letter] = [(a, b) for (a, b) in coord_list if b > x_line] + [
+                    (a, b + 1) for (a, b) in coord_list if b < x_line
                     ]
 
+    def flasher(self, full_lines, fill):
+        '''
+        Takes a list of full x lines and a color. Paints the blocks in the x_line with a given color.
+        Used in conjuction with full_line clear for flashing purposes
+        '''
+        for x in range(game_width):
+            for x_line in full_lines:
+                self.canvas.create_rectangle(
+                    x * square_size,
+                    x_line * square_size,
+                    x * square_size + square_size,
+                    x_line * square_size + square_size,
+                    tags="flash",
```

Comment:
The rest of the program doesn't use `flash` tag anywhere, but there is `self.canvas.delete("block")`:

```suggestion
                    tags="block",
```